Sources/FeaturevisorSDK/Instance+Variable.swift

```diff
@@ -72,25 +72,19 @@ extension FeaturevisorInstance {
             as? [String]
     }
 
-    public func getVariableObject<T: Decodable>(
+    public func getVariableObject(
         featureKey: FeatureKey,
         variableKey: String,
         context: Context
-    ) -> T? {
+    ) -> VariableObjectValue? {
```

Comment:
JS SDK has it as a generic though: https://github.com/featurevisor/featurevisor/blob/main/packages/sdk/src/instance.ts#L1240

Reply:
Yea, let's leave it as it is for Swift SDK as well. Closing this PR.